lib/StreamingStore.ts

```diff
@@ -75,9 +75,9 @@ implements RDF.Source<Q>, RDF.Sink<RDF.Stream<Q>, EventEmitter> {
         quad.graph,
       )) {
         for (const pendingStream of this.pendingStreams.getPendingStreamsForQuad(quad)) {
+          pendingStream.emit('quad', quad);
```

Comment:
I changed it here so that even when the match stream is not read with for instance a `data` event we can still get the `quad` event.

Reply:
Why is this change needed?

Reply:
because without it the `quad` event seems to only be produced when the stream is read, because it only produced this event when the pending stream is activated and it is only activated when the stream is read.

Reply:
I've added a comment to justify it.